backend/core/Engine.py

```diff
@@ -82,20 +82,29 @@ def __init__(self, jobFileName: str, thresholdsFileName: str, concurrentConnecti
             concurrentConnections = 50 if concurrentConnections is None else concurrentConnections
         AsyncioUtils.init(concurrentConnections)
 
+        self.password_dynamically = password_dynamically
```

Comment:
We don't need to save the password to the self context, I think this line can be removed